tests/test_connect.py

```diff
@@ -45,6 +46,13 @@
 CLIENT_SSL_KEY_FILE = os.path.join(CERTS, 'client.key.pem')
 CLIENT_SSL_PROTECTED_KEY_FILE = os.path.join(CERTS, 'client.key.protected.pem')
 
+if _system == 'Windows':
+    DEFAULT_GSSLIB = 'sspi'
+    OTHER_GSSLIB = 'gssapi'
```

Comment:
Since this makes it possible to use `gssapi` on Windows also, should the `gssapi` dependency be made unconditional of `platform_system`?

Reply:
Good question. My idea here was to only install the default library which is the most likely to work. Using `gssapi` on Windows requires installing Kerberos for Windows, so it probably needs manual steps anyway. We can install `gssapi` and let the user deal with the error for missing KfW, or we can document this better somewhere. Happy to do either way.

Technically, sspilib [can be used on non-Windows as well](https://github.com/jborean93/sspilib?tab=readme-ov-file#non-windows-support), so we can install both unconditionally, but this is even more experimental and exotic.